.github/workflows/tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.7, 3.8]
```

Comment:
I think we can include 3.6 if we prefetch numpy in the following section:
```patch
python -m pip install --upgrade pip
python -m pip install flake8 flake8-docstrings pytest
+ python -m pip install numpy
if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Reply:
Maybe replacing
```patch
- numpy >=1.18, <2.0
+ numpy >=1.17, <2.0
```
in requirements.txt will suffice?

Reply:
I think, since we cannot rely on the exact order in requirements, we have to install numpy first. This might not necessarily bound by a specific version because the requirements entry is read eventually. But it prevents some third party dependency to pull incompatible versions.

Reply:
Seems like this does the trick.